uis/MainWindow.py

```diff
@@ -3,35 +3,30 @@
 # Form implementation generated from reading ui file 'MainWindow.ui',
```

Comment:
The UI should have a checkbox (previously selected) that the user chooses to run the PCA on the multi-band raster (default value: True) variable name self.run_pca)

Reply:
Out of scope here, will save fo later